File: DataSummitSolution/DataSummitModels/DB/EmployeeTerritory.cs
﻿using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class EmployeeTerritory
    {
        public int EmployeeId { get; set; }
        public string TerritoryId { get; set; }
        public DateTime? CreatedDate { get; set; }
        public long? UserId { get; set; }
    }
}

File: DataSummitSolution/DataSummitUI/src/app/efModels/employeeTerritory.ts

export class EmployeeTerritory {

	EmployeeId: number;
	TerritoryId: string;
	CreatedDate?: Date | string;
	UserId?: number;

	constructor(employeeId?: number, territoryId?: string, createdDate?: Date | string,
		userId?: number)
	{
		this.EmployeeId = employeeId || 0;
		this.TerritoryId = territoryId || "";
		this.CreatedDate = createdDate || new Date(Date.now()) || null;
		this.UserId = userId || 0 || null;
	}
}
